authors/apps/core/utils/user_management.py

```diff
@@ -0,0 +1,22 @@
+import jwt
```

Comment:
@brucemakallan We already have code doing this. Please confirm

Reply:
Retrieving content from a token was used in many different areas so I refactored it to be in just one place I can call it from. It's the method that uses the `import jwt`